src/game/features/self/SuperPunch.cpp

```diff
@@ -0,0 +1,23 @@
+#include "core/commands/LoopedCommand.hpp"
+#include "game/features/Features.hpp"
+#include "game/rdr/Natives.hpp"
+
+namespace YimMenu::Features
+{
+	class Superpunch : public LoopedCommand
```

Comment:
#117 Aims to already implement a similar feature, although not identical. The PR does predate this one, yet I do prefer to have the feature on the other PR separated much like this one.

Reply:
> #117 Aims to already implement a similar feature, although not identical. The PR does predate this one, yet I do prefer to have the feature on the other PR separated much like this one.

Okay, a viable solution could be remove the line in #117 for the melee and then they will still have their feature and this will be separated properly.